core/controllers/admin.py

```diff
@@ -94,6 +95,12 @@ def get(self):
             (unicode(ind), exp[0]) for ind, exp in
             enumerate(feconf.DEMO_EXPLORATIONS)]
 
+        demo_collection_ids = [
```

Comment:
Sgtm. Changed feconf.DEMO_EXPLORATIONS to be a dict based on keys rather than a list.